### ISSUE TITLE
FileNotFoundError in Cleaning stage for absolute paths

### ISSUE THREAD

**Issue by CarMoreno** [SvenMarcus/hpc-rocket#52]
Hi 👋,

I have been using this amazing tool for a couple of months and It's been really useful in my job! However, I think I found a little bug 🐛. If you set the folders/files using an absolute path for the cleaning stage, they won't be found.

For example, this config won't work:

```yml
clean:
  - /path/to/folder
  - /path/to/file.txt
```

Let me know if I am doing something wrong or if it is a proper bug.

Thank you!

**Comment by CarMoreno** [comment]
Friendly ping @SvenMarcus :D

**Comment by SvenMarcus** [comment]
Glad to hear that it turned out to be useful! 😄
I released another version that adds compatibility for the Python versions 3.9 up to 3.13

If you are interested in contributing further, let me know. At my workplace i have very little time to improve or even maintain hpc-rocket and most of the development has to happen in my free time.
So any kind of contribution is very much appreciated!

**Comment by CarMoreno** [comment]
Yes @SvenMarcus , I'm really interested to help. I just check the compatibility with the other python versions, thanks for that!
